bin/ember-template-lint.js

```diff
@@ -77,7 +77,12 @@ function expandFileGlobs(positional) {
         ignore: ['**/dist/**', '**/tmp/**', '**/node_modules/**'],
         gitignore: true,
       })
-      .filter(filePath => filePath.slice(-4) === '.hbs')
+      .filter(
+        filePath =>
+          filePath.slice(-4) === '.hbs' ||
```

Comment:
> no need to slice, since we aren't really using the result


I think there is no difference between this `filePath.slice(-4) ===  '.hbs'` and this `filePath.endsWith('.hbs')`, **slice** will **not modify** the original string **filePath**.

But here, i prefer using `endsWith` for readability reason :)

Reply:
> I think there is no difference between this filePath.slice(-4) === '.hbs' and this filePath.endsWith('.hbs'), slice will not modify the original string filePath.

Yep, I totally understand this is how `.slice` works. I just don't think we need `.slice` because we aren't using the result and we want **exactly* what `.endsWith` is made for 😸